<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>De</class>
 <widget class="QWidget" name="De">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Simulation Dé</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="buttonAfficherValeur">
     <property name="text">
      <string>affiche la valeur du dé</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="buttonTirerDe">
     <property name="text">
      <string>tire le dé</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelValeurDe">
     <property name="text">
      <string>Valeur du dé : 0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="buttonScoreTotal">
     <property name="text">
      <string>score total</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="buttonRemettreZero">
     <property name="text">
      <string>remise à zéro du score</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelScoreTotal">
     <property name="text">
      <string>Score total : 0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
